content/terms/how-to/apply-filters.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Apply filters
+weight: 7
+---
+
+# Apply filters
+
+This guide explains how to apply filters to existing declarations to remove meaningless content that changes on each page load or that cannot be removed with CSS selectors to avoid noise in the terms changes history.
+
+## Prerequisites
+
+- An existing terms declaration file
+- Identified the noise you want to remove and ensure it cannot be removed with CSS selectors with the [`remove`]({{< relref "terms/reference/declaration/#ref-remove" >}}) property.
+
+## Step 1: Check for built-in filters
```

Comment:
I think I'll use a structure based on the principle that we most often use a built-in filter and optionally a custom filter, so I would put everything related to creating a custom filter on a dedicated how-to page.